Server/src/database.ts

```diff
@@ -1,7 +1,8 @@
 import { createConnection, getManager } from 'typeorm';
+import 'dotenv/config';
 
-export const connectDB = async () => {
-  await createConnection().then(async connection => {
+export const connectDB = async (config) => {
```

Comment:
i suggest that if we dont know type, at least put ':any', imo that way we enforce at least some typing and make it a habit to add types. Of course ideally we would know the type of the config and put it there, is it just a string?